packages/client/src/object/convertWireToOsdkObjects/createOsdkObject.ts

```diff
@@ -149,7 +150,13 @@ export function createOsdkObject<
               target[RawObject][objectDef.primaryKeyApiName as string],
               p as string,
               rawValue.type === "geotimeSeriesValue"
-                ? { time: rawValue.timestamp, value: rawValue.position }
+                ? {
```

Comment:
How could I have avoided this(besides triple checking this)? Is there a better practice here when dealing with any's?

Reply:
You can do `{ foo: "bar" } satisfies BlahInterface` as an extra check